editor/editor_node.cpp

```diff
@@ -2544,6 +2540,27 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 	}
 }
 
+void EditorNode::_tool_menu_option(int p_idx) {
+	if (p_idx == 0) {
+		orphan_resources->show();
```

Comment:
I'm not fond of hardcoding what tool options do based on magic numbers from their inserting order.

The previous system let the item register itself as `TOOLS_ORPHAN_RESOURCES`, the new system should stay similar. The menu item should contain the necessary information to match it to its behaviour, not its index.